docs/zh-cn/2.0.2/user_doc/architecture/design.md

```diff
@@ -1,116 +1,67 @@
-## 系统架构设计
-本章节介绍Apache DolphinScheduler调度系统架构
+架构概念
+=============================
 
+## 架构概述
 
-### 1.系统架构
+DolphinScheduler 是一个分布式易扩展、去中心化的可视化DAG工作流任务调度系统，以DAG流式的方式将Task组装起来，可实时监控任务的运行状态，同时支持执行重试、kill等任务。
 
-#### 1.1 系统架构图
 <p align="center">
   <img src="/img/architecture-1.3.0.jpg" alt="系统架构图"  width="70%" />
   <p align="center">
         <em>系统架构图</em>
   </p>
 </p>
 
-#### 1.2 启动流程活动图
-<p align="center">
-  <img src="/img/master-process-2.0-zh_cn.png" alt="Start process activity diagram"  width="70%" />
-  <p align="center">
-        <em>启动流程活动图</em>
-  </p>
-</p>
```

Comment:
If you delete img assert, you should delete the img file if without use in others place